modules/social_features/social_topic/social_topic.module

```diff
@@ -343,7 +343,7 @@ function social_topic_form_taxonomy_term_topic_types_form_alter(&$form, FormStat
     '#type' => 'inline_template',
     '#template' => '<div class="icon-svg-wrapper" data-href="#' . $default_icon . '"><svg class="selected-svg"><use xlink:href="#' . $default_icon . '"></use></svg>{{ svg_icons|raw }}</div>',
     '#context' => [
-      'svg_icons' => file_get_contents(drupal_get_path('theme', 'socialbase') . '/assets/icons/icons.svg'),
+      'svg_icons' => file_get_contents(\Drupal::service('extension.list.module')->getPath('socialbase') . '/assets/icons/icons.svg'),
```

Comment:
The same, use `\Drupal::service('extension.list.theme')->getPath('socialbase');`.